app/uk/gov/hmrc/emcstfe/models/request/chris/ChrisRequest.scala

```diff
@@ -28,4 +30,6 @@ trait ChrisRequest extends BaseRequest {
   def shouldExtractFromSoap: Boolean
 
   def metricName: String
+
+  val legacyCorrelationUUID: String = s"PORTAL${UUID.randomUUID().toString.replace("-","")}"
```

Comment:
Could we use the `correlationUUID` available in BaseRequest instead rather than creating a new one?

Reply:
Updated in https://github.com/hmrc/emcs-tfe/pull/94/commits/d99e48f4517592afc7047d7eabf58f9b2ae51f24